packages/vue3-vite/src/util/generate-test-target.ts

```diff
@@ -0,0 +1,25 @@
+import { TestFramework } from './test-framework';
+
+export const generateTestTarget = (
```

Comment:
Vitest and Jest are using different executors and options, so I added a helper function `generateTestTarget` that is used in both library generator and app generators.